### PR TITLE
Add ERC: Exponential Curves

### PR DESCRIPTION
**Pull request by 0xneves**
This ERC provides a function called `expcurve` that calculates the curve’s decay value at a given timestamp based on the initial timestamp, final timestamp, curvature, and curve direction (ascending or descending).

Discussion: https://ethereum-magicians.org/t/erc-xxxx-exponential-curves/20170

### CODE REVIEW

File: ERCS/erc-TBA.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-TBA.md:1]:
```suggestion
---
eip: 7725
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: ERCS/erc-TBA.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+title: Exponential Curves
+description: A dynamic model for exponential curves to handle various time-based events
+author: Guilherme Neves (@0xneves)
+discussions-to: https://ethereum-magicians.org/t/erc-xxxx-exponential-curves/20170
```
Comment (abcoathup) [inline ERCS/erc-TBA.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7725-exponential-curves/20170
```

**Comment by 0xneves** [comment]
Hey @SamWilsn I'm looking forward to your feedback!! I'll attend the next office hours  💯

**Comment by SamWilsn** [comment]
Thanks for the contribution! I think this is a better fit for a library instead of a standard, and might be better as a part of OpenZeppelin (or similar library.)

**Pull request closed without merging**